frontend/src/api/network/inferenceServices.ts

```diff
@@ -2,7 +2,15 @@ import { k8sGetResource, k8sListResource } from '@openshift/dynamic-plugin-sdk-u
 import { InferenceServiceModel } from 'api/models';
 import { InferenceServiceKind } from 'k8sTypes';
 
-export const listInferenceServices = (namespace: string): Promise<InferenceServiceKind[]> => {
+export const listInferenceService = (): Promise<InferenceServiceKind[]> => {
+  return k8sListResource<InferenceServiceKind>({
+    model: InferenceServiceModel,
+  }).then((listResource) => listResource.items);
+};
+
+export const getNamespacedInferenceServices = (
```

Comment:
No worries! I will change it in my next PR to set `namespace` as an optional prop because we may need to list the cluster InferenceServices for the global page and the namespace ones for the DSG details page.

Reply:
nice!